HybridLwcTemplate/server/force-app/main/default/aura/LightningDependencyApp/LightningDependencyApp.app-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AuraDefinitionBundle xmlns="urn:metadata.tooling.soap.sforce.com" fqn="LightningDependencyApp">
+    <apiVersion>46.0</apiVersion>
```

Comment:
Could this be auto-pulled from the SDK's `apiVersion`?

Reply:
We have never done that for our js assets (e.g. https://github.com/forcedotcom/SalesforceMobileSDK-Shared/blob/dev/libs/force.js).
We typically just move up manually once in a while after verifying that there is no behavior change in the new version.